modules/setting/mirror.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package setting
+
+import (
+	"time"
+
+	"code.gitea.io/gitea/modules/log"
+)
+
+var (
+	// Mirror settings
+	Mirror = struct {
+		Enabled         bool
+		DisableNewPull  bool
+		DisableNewPush  bool
+		DefaultInterval time.Duration
+		MinInterval     time.Duration
+	}{
+		Enabled:         true,
+		DisableNewPull:  false,
+		DisableNewPush:  false,
+		MinInterval:     10 * time.Minute,
+		DefaultInterval: 8 * time.Hour,
+	}
+)
+
+func newMirror() {
+	if err := Cfg.Section("mirror").MapTo(&Mirror); err != nil {
```

Comment:
Hmm... shouldn't we set

```suggestion
  Mirror.Enabled = !Cfg.Section("repository").Key("DISABLE_MIRRORS").MustBool(false)
	if err := Cfg.Section("mirror").MapTo(&Mirror); err != nil {
```